client/src/pages/Rental/Rent.js

```diff
@@ -245,7 +245,7 @@ const Rent = () => {
         className={classes.gridContainer}
       >
         {CarList.map((car) => (
-          <Grid item key={car.name} xs={12} sm={3}>
+          <Grid item key={car.name} xs={12} sm={3} md={3}>
             <Paper elevation={3} className={classes.paperContainer}>
```

Comment:
You can remove the 'md{3}'. On screen sizes small and up, it is already set to 3 columns.